packages/terra-button-group/package.json

```diff
@@ -27,11 +27,13 @@
   "peerDependencies": {
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
-    "terra-mixins": "^1.0.0"
+    "terra-mixins": "^1.0.0",
+    "terra-button": "^0.1.0"
```

Comment:
For components < 1.x, we have been versioning it like this:
```
"terra-button": "^0.x"
```
Which allows terra-button 0.1, 0.2.

This site is really useful for learning more about npm versioning:
https://semver.npmjs.com/